fennel/datasets/datasets.py

```diff
@@ -507,10 +519,19 @@ def aggregate(self, *args, **kwargs) -> _Node:
         if len(self.keys) == 1 and isinstance(self.keys[0], list):
             self.keys = self.keys[0]  # type: ignore
 
+        if along is not None and not isinstance(along, str):
+            raise ValueError(
+                f"Along argument have to be string or None got: {along}"
```

Comment:
improve the error:

`along kwarg in the aggregate operator must be string or None, found: `{along}``